documentation/general/dash-sonic-hld.md

```diff
@@ -1008,7 +1075,7 @@ For the example configuration above, the following is a brief explanation of loo
 		g. Underlay DIP shall be 50.1.2.3 (from mapping), Underlay SIP shall be 55.1.2.3 (from ENI)
```

Comment:
On e.3 (3 lines above), since both src and dst are CA addresses, we should call out that it is a source CA

Reply:
addressed